SunEngine/InfoPrinter.cs

```diff
@@ -5,20 +5,20 @@
 
 namespace SunEngine
 {
-    public static class InfoPrinter
```

Comment:
Static won't be testable in future.

Reply:
I thing this is good practice not in global case but in case like this. This class just show message. I do not see herd mocs or some tests, and even if they are don't see hardness in that case.

Reply:
It does not only show the message, but it also does IO operation - it's considered slow and should not be executed in UnitTests. Usually, static calls used for very lightweight libraries and operations (calculations, validations etc.)

Reply:
Mabe to move text to code?
Not to allow class to be static. But I am not sure that separate file is good idea.
I asked minute ago. In C# we can make multiline texts easy human read as
```
string str = @"Hi
I am multiline
text";

```